recipes/h5pp/all/conanfile.py

```diff
@@ -40,6 +38,11 @@ def _configure_cmake(self):
             self._cmake.configure(source_folder=self._source_subfolder)
         return self._cmake
 
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "17")
```

Comment:
Ok now some triplets will fail in CCI because compiler doesn't support C++17. You can use this pattern: https://github.com/conan-io/conan-center-index/blob/86049562aa959d0e5774799c3d80f38527ed54c6/recipes/spy/all/conanfile.py#L28

Reply:
What is a triplet? Thanks for the suggestion, I'll use the pattern.